### DISCUSSION TITLE
看了源码，发现sft构造的训练样本都是单轮对话，训练好后可以支持多轮对话吗？

### DISCUSSION THREAD

**Discussion by minmie** [ymcui/Chinese-LLaMA-Alpaca-2#301] [Q&A]
如下图，是训练时候构造的输入格式，看起来都是单轮的（图一）
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/40080081/4527dbd7-c5ea-4adc-8754-3eedc7b36104)

看了相关文章，llama2-chat多轮的prompt如下（图二）
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/40080081/0cd842e4-c5b8-4ac0-9d06-4702c5350e3e)

想请问下，
1. 训练的时候都是单轮，推理的时候可以支持多轮？
2. 推理的时候多轮对话怎么构造prompt?是基于图二吗（可是我们训练的时候用的都是图一的prompt，推理的时候直接用图二的prompt不会有问题吗？）？

**Comment by airaria** [comment]
See #303
